src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -278,15 +281,59 @@ class CaseNotesReconciliationService(
       legacyId = caseNoteId,
     )
   }
+}
+
+private val algorithmChangeDate = LocalDateTime.parse("2025-07-17T08:08:00")
+
+private fun String.truncateToFitNomis(amendmentDateTime: LocalDateTime): String = if (
```

Comment:
is this the date time of the amendment or the last modified date time of the case note?